src/data/externalTutorials.json

```
@@ -1,4 +1,32 @@
 [
+  {
+    "url": "https://medium.com/cyfrin/what-is-eip-4844-proto-dankshading-and-blob-transactions-explained-984dec50dcc8",
+    "title": "What is EIP-4844? Proto-Danksharding and blob transactions explained",
+    "description": "What is the EIP-4844? Learn what proto-danksharding and blobs are, how they work, and how to send your first blob transaction using the new Ethereum improvement proposal",
+    "author": "Patrick Collins",
+    "authorGithub": "https://github.com/PatrickAlphaC",
+    "tags": [
+      "layer 2"
+    ],
+    "skillLevel": "intermediate",
+    "timeToRead": "660",
```

Is this a mistake?

---

Nope! It's an 11 minute read, so that's 660 seconds right? .... or that should be 11 then huh.

---

These should be in minutes 🙂 

```suggestion
    "timeToRead": "11",
```